core/src/main/java/dk/alexandra/fresco/framework/util/DrngImpl.java

```diff
@@ -48,6 +51,19 @@ public BigInteger nextBigInteger(BigInteger limit) {
     return result.compareTo(limit) < 0 ? result : nextBigInteger(limit);
   }
 
+  @Override
+  public boolean nextBit() {
+    if (bitsLeft == 0) {
+      drbg.nextBytes(randomBytes);
+      bitsLeft = RANDOMBUFFER_SIZE * Byte.BYTES * 8;
```

Comment:
```suggestion
      bitsLeft = RANDOMBUFFER_SIZE * Byte.SIZE;
```
`Byte.Bytes * 8` seems a little silly :)